source/index.rst

```diff
@@ -1,16 +1,35 @@
 .. SPDX-FileCopyrightText: 2020 Hyperledger
    SPDX-License-Identifier: CC-BY-4.0
-   
-Perun \@  Hyperledger Labs
-==========================
 
-A blockchain-agnostic state channels framework
+.. image:: images/perun_logo.svg
+   :width: 300
+   :alt: Perun Logo
+   :align: center
+
+|
+
+The Perun Framework
```

Comment:
@matthiasgeihs  What are your thoughts on merging the `Introduction` section in `introduction` section here. Because both of the sections seem to target the same purpose of providing first glance introduction to the project.

The `introduction.rst` page will then begin with the `Perun Protocol` as its first section.

Reply:
I thought about it too. But it is some work to do it right. If we move the introduction part from there, you may want to write a different header for that part and potentially reorganize the section a bit accordingly. For example, I could see that we merge "Introduction" and "Concepts" into one larger section.

Anyhow, I think applying such structural changes is some work and I would propose to do it in another PR.